config/sshdcond/sshdcond.inc

```diff
@@ -117,23 +117,26 @@ function sshdcond_custom_php_write_config() {
 function sshdcond_sync_on_changes() {
 	global $config, $g;
 
-	if (is_array($config['installedpackages']['sshdcondsync'])) {
-		if (!$config['installedpackages']['sshdcondsync']['config'][0]['synconchanges']) {
-			return;
-		}
-	}
-
-	log_error("[sshdcond] xmlrpc sync is starting.");
-	foreach ($config['installedpackages']['sshdcondsync']['config'] as $rs) {
-		foreach($rs['row'] as $sh) {
-			$sync_to_ip = $sh['ipaddress'];
-			$password = $sh['password'];
-			if ($password && $sync_to_ip) {
-				sshdcond_do_xmlrpc_sync($sync_to_ip, $password);
+	if (!is_array($config['installedpackages']['sshdcondsync'])) {
+		/* Basically, this package was never configured */
+		return;
+	} elseif (!$config['installedpackages']['sshdcondsync']['config'][0]['synconchanges']) {
```

Comment:
Actually you don't need elseif here, because if it entered in above if, it's going to return. You also don't need that else on line 126. You will end up with something like:
```
if (!is_array(...)) {
    return;
}
if (!isset(...)) {
    return;
}
/* Do XMLRPC sync */
```